workflow-controller/controller-service/src/main/java/io/hyscale/controller/Converters/ProfileConverter.java

```diff
@@ -46,33 +47,32 @@ public String getFilePattern() {
 
 
     @Override
-    public String getReferenceSchema() {
-        return PROFILE_SCHEMA_FILE;
+    public HyscaleSpecType getReferenceSchemaType() {
+        return HyscaleSpecType.ENVIRONMENT;
     }
 
     @Override
-    public String getWarnMessage() {
-        return ValidationConstants.INVALID_PROFILE_FILE_NAME_MSG;
+    public ServiceSpecActivity getWarnMessage() {
```

Comment:
used anywhere ? need to be removed i guess..please confirm

Reply:
yes its used in converter.